### PR TITLE
Fix firstrunwizard layout issues

### PR DESCRIPTION
**Pull request by juliushaertl**
For https://github.com/nextcloud/server/issues/16076

 - [x] First run wizard should have different viewer background opacity (less dark)
 - [x] Scrollbar shows (only on very small screen/high zoom level)
 - [x] Container shouldn’t need to resize between slides, adjust content, fix height and width
 - [x] Content should fit into box 
  @jancborchardt We should consider stipping out some of the content of the last page probably.
 It now works also with 150% zoom level on a full hd screen. I think its fine to have the scroll bar for higher zoom levels.

Peek 2019-06-27 23-37

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -241,10 +239,42 @@
 </style>
 
 <style lang="scss" scoped>
-	// modal layout
+
+	.modal-mask {
+		background-color: rgba(0, 0, 0, 0.7);
+
+		& /deep/ .modal-wrapper .modal-container {
```
Comment (skjnldsv) [inline src/App.vue]: better use v-deep alias as /deep/ will not be transpiled and is not supported in all platforms ;)

File: src/App.vue [hunk 2]
```diff
@@ -241,10 +239,42 @@
...
+			display: flex;
+			flex-direction: column;
+			height: 95% !important;
+			width: 95% !important;
+			max-width: 900px;
+			max-height: 650px !important;
+			position: relative;
+		}
+
+		& /deep/ .modal-body {
```
Comment (skjnldsv) [inline src/App.vue]: You're editing the default styling, but maybe the component needs a fix?
Did you find anything that was maybe weird or unfit in the component itself? :)

**Comment by jancborchardt** [comment]
I’m out towards vacation, so I’m not available here – sorry. :\

**Comment by karlitschek** [comment]
looks nice 👍

File: src/App.vue (hunk 1 above)
Comment (juliushaertl) [reply src/App.vue]: @skjnldsv Do you have a reference for that? vue-loader docs just list /deep/ and >>> https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors but >>> is a scss selector

I couldn't find them in the bundled files, so I assume they are transpiled just fine

File: src/App.vue (hunk 2 above)
Comment (juliushaertl) [reply src/App.vue]: Fixed.

File: src/App.vue (hunk 1 above)
Comment (skjnldsv) [reply src/App.vue]: https://github.com/vuejs/vue-loader/issues/913

Comment (skjnldsv) [reply src/App.vue]: Issue is that /deep/ was not transpiled as this was a soon to be added css property. So it stayed this way. But it's now removed

Comment (skjnldsv) [reply src/App.vue]: https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors

Comment (juliushaertl) [reply src/App.vue]: Ah thanks. :+1:

File: src/App.vue [hunk 3]
```diff
@@ -344,24 +373,22 @@ export default {
 			return this.currentSlide === 0
 		}
 	},
-	beforeMount() {
-		axios.get(OC.generateUrl('/apps/firstrunwizard/wizard')).then((response) => {
-			this.slides = response.data
-			this.showModal = true
-		})
-
+	created() {
```
Comment (skjnldsv) [inline src/App.vue]: :+1:

**Pull request merged**